### PR TITLE
Implement basic control and monitor

### LINKED ISSUE #4: Basic control and monitor
### Feature you want to implement
I will implement to control or monitor the basic settings except to reading a wavelength.

### How the feature is implemented
Settings to be implemented:
- Start or stop measurement.
- Set or get the exposure time.
- Set or get the active channel.
- ~~Set or get the measurement interval.~~
- ~~Initialize the calibration.~~
- ~~Get all settings.~~

### Additional context
See also: [PyLabLib wlm docs](https://pylablib.readthedocs.io/en/latest/.apidoc/pylablib.devices.HighFinesse.html#module-pylablib.devices.HighFinesse.wlm)

### PR DESCRIPTION
**Pull request by BECATRUE**
This closes #4.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -62,3 +62,52 @@ async def lifespan(_app: FastAPI):
...
+
+
+@app.post("/measurement/stop/")
+async def stop_measurement():
+    """Stops the measurement."""
+    wlm.stop_measurement()
+
+
+@app.post("/exposure/set/")
+async def set_exposure(channel: int, exposure: float):
```
Comment (kangz12345) [inline main.py:80]: Let's denote the unit in the name of the argument, e.g. `exposure_s`.

File: main.py [hunk 2]
```diff
@@ -62,3 +62,52 @@ async def lifespan(_app: FastAPI):
...
+    
+    Args:
+        channel: Target channel.
+        exposure: Target exposure.
+    """
+    wlm.set_exposure(exposure=exposure, channel=channel)
+
+
+@app.get("/exposure/get/")
+async def get_exposure(channel: int) -> float:
```
Comment (kangz12345) [inline main.py:91]: I'm not sure what is the best way to specify the unit of the exposure time.

**Review by kangz12345 — CHANGES_REQUESTED** [review]

File: main.py (hunk 2 above)
Comment (BECATRUE) [reply main.py:91]: How about specify the unit in the method name, e.g., `get_exposure_s()`? @kangz12345

Comment (kangz12345) [reply main.py:91]: That sounds good!

**Comment by BECATRUE** [comment]
We decided to implement a WLM server in the `wlm-server` repository, hence I close this PR.

**Pull request closed without merging**